recipes/vir-simd/all/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.15)
+project(test_package LANGUAGES CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
```

Comment:
This can be removed, conan-center-index no longer require ``test_v1_package`` (it doesn't harm to have them there, you can leave it, just know it is no longer a requirement)

Reply:
Sorry I've just seen your comments and it's already merged. I'll keep them in mind for the next update.